package.json

```diff
@@ -41,7 +89,8 @@
     "pretest": "npm run compile && npm run lint",
     "lint": "eslint src --ext ts",
     "test": "node ./out/test/runTest.js",
-    "package": "vsce package"
+    "package": "vsce package",
+    "cpi": "npm run compile && npm run package && code-insiders --install-extension $(ls context-copy-*.vsix | head -n 1)"
```

Comment:
Why this?

Reply:
Shorthand for quickly compiling, packaging, then installing on my local to test the extension out in vscode insiders. I can remove it though and set up a private alias should you want me to do so.